src/libasr/codegen/asr_to_llvm.cpp

```diff
@@ -1541,8 +1541,9 @@ class ASRToLLVMVisitor : public ASR::BaseVisitor<ASRToLLVMVisitor>
     }
 
     void visit_DictItem(const ASR::DictItem_t& x) {
-        ASR::Dict_t* dict_type = ASR::down_cast<ASR::Dict_t>(
-                                    ASRUtils::expr_type(x.m_a));
+        ASR::Dict_t *dict_type = ASR::down_cast<ASR::Dict_t>(
+                                        ASRUtils::expr_type(x.m_a));
+
```

Comment:
Similarly here. Ideally, there should be no formatting changes unless they actually improve something.